examples/inja/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2016-2017, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-inja)
+
+set(CMAKE_CXX_STANDARD 11)
+if(WIN32 AND MSVC AND MSVC_VERSION LESS 1900)
```

Comment:
Add this check to `hunter.cmake` and call `hunter_report_broken_package`. This will report error before build starts:
* https://github.com/ruslo/hunter/blob/46986a7de95398264e0bca290742296b2143a3a4/cmake/projects/Boost/mpi/hunter.cmake#L10-L15

Reply:
The problem really originates from the JSON package. I'll remove it here and file a separate PR that fixes 

* https://github.com/ruslo/hunter/blob/46986a7de95398264e0bca290742296b2143a3a4/cmake/projects/nlohmann_json/hunter.cmake#L7-L9